### PR TITLE
composefs: remove cgo dependency

### PR DESCRIPTION
**Pull request by giuseppe**

### CODE REVIEW

**Review by cgwalters — APPROVED** [review]

File: pkg/loopback/loop_wrapper.go [hunk 1]
```diff
@@ -34,19 +20,19 @@ type loopInfo64 struct {
...
-	LoopGetStatus64 = C.LOOP_GET_STATUS64
-	LoopSetStatus64 = C.LOOP_SET_STATUS64
-	LoopClrFd       = C.LOOP_CLR_FD
-	LoopSetCapacity = C.LOOP_SET_CAPACITY
+	LoopSetFd       = 0x4C00
+	LoopCtlGetFree  = 0x4C82
+	LoopGetStatus64 = 0x4C05
+	LoopSetStatus64 = 0x4C04
+	LoopClrFd       = 0x4C01
+	LoopSetCapacity = 0x4C07
```
Comment (mtrmac) [inline pkg/loopback/loop_wrapper.go:28]: A drive-by comment: Did you consider using `x/sys/unix` instead?
